extensions/wikia/HTTPSSupport/HTTPSSupportHooks.class.php

```diff
@@ -68,4 +70,10 @@ private static function httpsEnabledTitle( Title $title ): bool {
 		return array_key_exists( $wgDBname, self::$httpsArticles ) &&
 			in_array( $title->getPrefixedDBKey(), self::$httpsArticles[ $wgDBname ] );
 	}
+
+	public static function parserUpgradeVignetteUrls ( string &$url ): bool {
```

Comment:
This is no longer returning a bool, so the type hint on the return value should be removed.

Reply:
fix: 8e855226164